web/templates/tooltip_templates.hbs

```diff
@@ -117,7 +117,7 @@
     {{tooltip_hotkey_hints "D"}}
 </template>
 <template id="show-all-pms-template">
-    {{t 'All direct messages' }}
+    {{t 'DM feed' }}
```

Comment:
So this rename is incomplete; `git grep 'All direct messages'` shows you don't also update the help center.